OWNERS_ALIASES

```diff
@@ -66,14 +65,13 @@ aliases:
     - reedcort
     - mrWinston
   srep-functional-leads:
-    - mrbarge
     - rafael-azevedo
     - iamkirkbater
     - bng0y
     - tonytheleg
-    - karthikperu7
     - bmeng
     - mjlshen
+    - sam-nguyen7
```

Comment:
Need to add `ravitri` here as well I think

Reply:
Thanks Wenqi! This change will need to be part of boilerplate - https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator/OWNERS_ALIASES#L67 

I'll raise a PR for that change. Thanks to notice it.